hibernate-core/src/main/java/org/hibernate/query/Query.java

```diff
@@ -273,7 +273,7 @@ default Stream<R> stream() {
 	 * @param graph The graph to apply.
 	 * @param semantic The semantic to use when applying the graph
 	 *
-	 * @deprecated Use {@link #setEntityGraph(EntityGraph, GraphSemantic)}
+	 * @deprecated Use {@link SelectionQuery#setEntityGraph(EntityGraph, GraphSemantic)}
```

Comment:
Why did you make these changes?

Reply:
😖🙈 hmm, I thought I've reverted all of these (IDE made the change when I updated the method signature through "refactoring"  🤦🏻 ) will revert these